bser/decode.go

```diff
@@ -30,7 +30,7 @@ func (d *Decoder) Decode(dest interface{}) error {
 		return fmt.Errorf("Expected %x, found %x", protocolPrefix, buf)
 	}
 
-	var size int
+	var size int32
```

Comment:
is this in the spec?  if not, I'd rather leave it more generic.

Reply:
good call, looks to be specific to the python client.